<ui version="4.0" >
 <class>MixdownFileDialogBase</class>
 <widget class="QDialog" name="MixdownFileDialogBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MusE: Set Mixdown Wavefile</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>11</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="5" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="TextLabel1" >
     <property name="text" >
      <string>File Path</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="TextLabel2" >
     <property name="text" >
      <string>Channel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="4" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLineEdit" name="editPath" />
     </item>
     <item>
      <widget class="QToolButton" name="buttonPath" >
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="../muse.qrc" >:/xpm/fileopen.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" >
    <widget class="QComboBox" name="comboChannel" >
     <item>
      <property name="text" >
       <string>Stereo</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Mono</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>5.1</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="4" >
    <widget class="QComboBox" name="comboFormat" >
     <item>
      <property name="text" >
       <string>wav,16 Bit</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>wav, 24 Bit</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>wav, 32 Bit (float)</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="3" >
    <widget class="QLabel" name="TextLabel3" >
     <property name="text" >
      <string>Format</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../muse.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>MixdownFileDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel" >
     <x>99</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>MixdownFileDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>318</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel" >
     <x>170</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
